storage/src/vespa/storageframework/defaultimplementation/clock/fakeclock.h

```diff
@@ -22,13 +22,12 @@ struct FakeClock : public framework::Clock {
 
 private:
     Mode _mode;
-    framework::MicroSecTime _absoluteTime;
+    vespalib::duration      _absoluteTime;
```

Comment:
Semantically it would make more sense to let this be a `time_point`, but it would require more wrapping and unwrapping...

Reply:
Agree, leaving as is.